mscore/data/shortcuts-Mac.xml

```diff
@@ -260,6 +260,10 @@
     <key>prev-segment-element</key>
     <seq>Ctrl+Alt+Shift+Left</seq>
     </SC>
+  <SC>
+    <key>get-location</key>
+    <seq>Ctrl+Alt+L</seq>
```

Comment:
I am not sure about Windows an Mac, but on Linux Ctrl+Alt+L is a common shortcut which turns on a screensaver, and Ctrl+Alt shortcuts in general tend to be reserved by system-global shortcuts. Maybe something like Ctrl+Shift+L (seems to be used by "lock score" currently) or Alt+L could work better? Although I am not sure what is the general rule for choosing default shortcuts currently.

Reply:
There's not a lot of great choices left, but "lock score" isn' even a supported command as far as I know.  Doesn't work, it just disables menus etc but still leaves drag working, which is really the main reason people want a lock function.  So I propose stealing that.

FWIW, I did try to get feedback on Telegram regarding general rules, but didn't really get much response.  Historically, I know it's been pretty haphazard.

In another PR I have a default shortcut added for palette search, Ctrl+Alt+P.  Ctrl+Shift+P is already used for plugin creator, and while I personally would be OK with stealing that, I'd rather not annoy plugin creators.  Ctrl+Shift+S is available, though.